pages/projusers.md

```diff
@@ -15,19 +15,25 @@ of the Scikit-HEP project.
 Experiment collaborations
 -------------------------
 
+[![img-ATLAS]][ATLAS]
+
+[ATLAS](https://atlas.cern/) - A Toroidal LHC ApparatuS at CERN, Switzerland.
```

Comment:
I think ATLAS is just "ATLAS" now and we've given up the ridiculous backronym. :)

```suggestion
[ATLAS](https://atlas.cern/) - the ATLAS experiment at CERN, Switzerland.
```

Reply:
Indeed I did not know if the "definition" of the acronym is still used these days. Happy to go with your educated suggestion :-).